torch/_inductor/runtime/halide_helpers.py

```diff
@@ -0,0 +1,148 @@
+# mypy: allow-untyped-defs
+try:
+    import halide as hl  # type: ignore[import-untyped, import-not-found]
+except ImportError:
+    hl = None
+
+PHILOX_N_ROUNDS_DEFAULT = 10  # Default number of rounds for philox
+PHILOX_KEY_A_U32 = hl.u32(0x9E3779B9)
```

Comment:
nit: only do this if hl is not None: to avoid crashing when halide is not available?